docs/advanced-topics/wes/wes-wdl-agc-tutorial.rst

```diff
@@ -1,5 +1,11 @@
-WES Tutorial for AWS AGC
-========================
+WES Tutorial for running WDL workflows through the Cromwell Engine on AWS AGC
+=============================================================================
+
+.. note::
+    Although the process for running CWL workflows through the Toil engine on AWS AGC is very similar to the process
+    described in this tutorial. There are a few differences. So, for information on how to launch a CWL workflow on AWS
```

Comment:
Same comment as above about although and a comma

Reply:
Fixed